uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -329,19 +329,19 @@ public void disableMultipleWpj() {
 
     /**
      * Changes flight provider to local flights provider and sets
-     * PRTv3 flight flag to true.
+     * transfer token generation flight flag to true.
      */
-    public void enablePrtV3() {
```

Comment:
I think some tests were using this method, did we remove those references before removing the method?

Reply:
I updated this PR by removing the usage https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/2187/files